boot/limine.cfg

```diff
@@ -1,6 +1,7 @@
 TIMEOUT=1
 GRAPHICS=no
-THEME_COLOURS=60000000;aa0000;00aaff;aa5500;0000aa;aa00aa;9076de;aaaaaa
+
```

Comment:
Any reason for the extra newline and slight change in theme when we don't enable graphics?

Reply:
Newline doesn't necessarily have a purpose. Color changes apply to the text and not just the graphics when the framebuffer is enabled.

Reply:
Ah cool. I didn't notice it, but that's nice.